src/Documentation/sidebar.json

```diff
@@ -8,19 +8,19 @@
       "initialize",
       "configure",
       "add-files",
-      "share-data",
+      "store-data",
```

Comment:
please, grep and update the links to this chapter if we have any

Reply:
I did look, no links to update (: